wit/messaging.wit

```diff
@@ -2,10 +2,19 @@ package wasi:messaging@0.2.0-draft;
 
 world imports {
 	import producer;
-	import consumer;
 }
 
-world messaging {
+world imports-request-reply {
 	include imports;
-	export messaging-guest;
-}
\ No newline at end of file
+	import request-reply;
+}
+
+world messaging-request-reply {
```

Comment:
suggest: add docs for each world in this package.

Reply:
Addressed ✅